neurom/io/hdf5.py

```diff
@@ -59,21 +58,20 @@ def get_version(h5file):
         return 'H5V2'
 
 
-(PX, PY, PZ, PD) = xrange(4)  # points
-(GPFIRST, GTYPE, GPID) = xrange(3)  # groups or structure
+POINT_DIAMETER = 3
+GPFIRST, GTYPE, GPID = xrange(3)  # groups or structure
 
 Section = namedtuple('Section', 'ids, ntype, pid')
 
 
 def read(filename, remove_duplicates=False, data_wrapper=DataWrapper):
-    '''Read a file and return a tuple of data, format.
+    '''Read a file and return a `data_wrapper'd` data
 
     * Tries to guess the format and the H5 version.
     * Unpacks the first block it finds out of ('repaired', 'unraveled', 'raw')
 
     Parameters:
-        remove_duplicates: boolean, \
-        If True removes duplicate points \
+        remove_duplicates: boolean, If True removes duplicate points
```

Comment:
Good to know, thanks for the input.

Let's take this discussion to #553, so I know exactly what I need to do.